src/models/sessionBus.js

```diff
@@ -23,12 +23,15 @@ const {GObject} = imports.gi;
 const {FlatpakSharedModel} = imports.models.shared;
 const {FlatsealOverrideStatus} = imports.models.overrideStatus;
 
+/* https://dbus.freedesktop.org/doc/dbus-specification.html */
+const EXP = /^(([A-Z]|[a-z]|[0-9]|_|-)+)(\.(([A-Z]|[a-z]|[0-9]|_|-)+))+(\.\*){0,1}$/;
```

Comment:
To make this better than I currently do it, how about we officially move the definition of this esoteric regexp to this model (e.g. `var BUS_NAME_REGEX = ...`), and then we import this constant here https://github.com/tchx84/Flatseal/blob/master/src/widgets/busNameRow.js#L28 (instead of duplicating it)